demo/qat-init.sh

```diff
@@ -3,7 +3,7 @@
 NODE_NAME="${NODE_NAME:-}"
 ENABLED_QAT_PF_PCIIDS=${ENABLED_QAT_PF_PCIIDS:-37c8 4940 4942 4944}
 DEVS=$(for pf in $ENABLED_QAT_PF_PCIIDS; do lspci -n | grep -e "$pf" | grep -o -e "^\\S*"; done)
-SERVICES_LIST="sym;asym dc"
+SERVICES_LIST="sym asym sym;asym dc sym;dc asym;dc dcc"
```

Comment:
`dcc` does not exist yet so let's leave it out for now:

```suggestion
SERVICES_LIST="sym asym sym;asym dc sym;dc asym"
```

Reply:
Sure. Your suggestion would remove too much, though.

Reply:
oops, yes. updated